### PR TITLE
DM-30053: Add faro git ref job parameter.

### PR DESCRIPTION
**Pull request by ktlim**

### CODE REVIEW

**Review by SimonKrughoff — APPROVED** [review]
I don't quite understand how all this works internally.  If you are sure this override is making it into the job, I'm glad it's that simple.

**Comment by ktlim** [comment]
Oops, sorry; didn't notice that I hadn't pushed the real changes!  Can you please re-review?

File: pipelines/sqre/validate_drp_gen3.groovy [hunk 1]
```diff
@@ -128,6 +131,7 @@ def void verifyDataset(Map p) {
     'slug',
```
Comment (SimonKrughoff) [inline pipelines/sqre/validate_drp_gen3.groovy:131]: Maybe add a line for the new `p.gitRef` in the doc strings for this function

**Review by SimonKrughoff — APPROVED** [review]
That looks much better :)

One minor comment about the doc string

**Pull request merged**